src/main/org/nlogo/nvm/LetBinding.java

```diff
@@ -4,9 +4,9 @@
 
 import org.nlogo.api.Let;
 
-strictfp class LetBinding {
-  final Let let;
-  Object value;
+strictfp public class LetBinding {
+  public final Let let;
+  public Object value;
```

Comment:
Well, that's embarrassing :P Could I just change this to a scala case class, making `value` a `var`? That gives us customizable accessors if we want them, with convenient syntax.